aws/s3/s3_integration_test.go

```diff
@@ -0,0 +1,541 @@
+//go:build localstack
+// +build localstack
+
+package s3
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	CustomAWSEndpointURL = "http://localhost:4566"
+	AWSRegion            = "ap-southeast-2"
+	TestBucket           = "test-bucket"
```

Comment:
If not exporting these constants (so the downstream applications can reference these) , then could you make these consts lowercase prefix? Thanks.